### PR TITLE
Added Click CLI, pytube for downloading audio stream of YouTube video

### LINKED ISSUE #4: Create CLI using Click
- [x]  Design general CLI for the user to interact with - project heading, menu, instructions, help etc.
- [x]  Take YouTube URL as input and download the video using `pytube`.
- [x]  Get local file URL and forward it for #1

### PR DESCRIPTION
**Pull request by m0mosenpai**
# Description

Created minimal CLI using Click. Used pytube to extract audio-stream from YouTube video and passed it on to the Speech-To-Text API for further processing.

## Related Issues

Closes #4 

## Type of Change

- [ ] Bug Fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist

- [x] The pull request is for branch `develop`
- [x] `black` gave 0 errors
- [x] `flake` gave 0 errors
- [x] Squashed commits into a single one

# Screenshots of working

image

### CODE REVIEW

File: clipbit.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+def welcome_message():
+    os.system("clear")
+    print(" ██████╗██╗     ██╗██████╗ ██████╗ ██╗████████╗")
+    print("██╔════╝██║     ██║██╔══██╗██╔══██╗██║╚══██╔══╝")
+    print("██║     ██║     ██║██████╔╝██████╔╝██║   ██║   ")
+    print("██║     ██║     ██║██╔═══╝ ██╔══██╗██║   ██║   ")
+    print("╚██████╗███████╗██║██║     ██████╔╝██║   ██║   ")
+    print(" ╚═════╝╚══════╝╚═╝╚═╝     ╚═════╝ ╚═╝   ╚═╝   \n")
```
Comment (gmelodie) [inline clipbit.py:69]: Awesome!!! :love_you_gesture:

**Review by gmelodie — APPROVED** [review]
LGTM

**Review by Miguel-Enrique13 — APPROVED** [review]
Everything looks great!

**Comment by KimaruThagna** [comment]
LGTM
Looks good

**Pull request merged**